ion/symbols_table_test.go

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ion
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"math/rand"
```

Comment:
You should set up the `goimport` file watcher in your IDE.  It will do formatting plus order your imports.

Reply:
It does sort the imports for me AFAIK, is there more to the configuration?

Reply:
Are you using the `-w` flag to ensure that it writes the result back to the file?